schema/schema/asset.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
```

Comment:
Is this file also part of this pull request? please remove files unrelated to logger